src/icons/src/IconWrapper.js

```diff
@@ -20,7 +21,16 @@ export const IconWrapper = memo(
       iconWithProps = React.cloneElement(icon, { ...props, ...icon.props, ref })
     }
 
-    return iconWithProps
+    return (
+      <Box
+        display="flex"
```

Comment:
I don't think we can use `flex` in every scenario. I think `inline-flex` is probably closer to the current behavior?